red/runtime/nodes/flows/index.js

```
@@ -592,7 +716,13 @@ function updateFlow(id,newFlow) {
         }
         label = activeFlowConfig.flows[id].label;
     }
-    var newConfig = clone(activeConfig.flows);
+    
+    if (settings.enableFlowSharding) {
+        var newConfig = activeConfig.flows.slice();
+    } else {
+        var newConfig = clone(activeConfig.flows);
```

Biome (2.1.2)

[error] Shouldn't redeclare 'newConfig'. Consider to delete it or rename it.

'newConfig' is defined here:


(lint/suspicious/noRedeclare)